src/test/java/kitchenpos/acceptance/TableAcceptanceTest.java

```diff
@@ -0,0 +1,135 @@
+package kitchenpos.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import kitchenpos.AcceptanceTest;
+import kitchenpos.TestApiClient;
+import kitchenpos.domain.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("테이블 관련 기능")
+public class TableAcceptanceTest extends AcceptanceTest {
+    private OrderTable 테이블1;
+    private Menu 메뉴;
+    private Product 소고기한우;
+    private MenuGroup 추천메뉴;
+    private Order 주문1;
+
+    @BeforeEach
+    public void setUp() {
+        super.setUp();
+
+        추천메뉴 = MenuAcceptanceTest.메뉴그룹_등록되어있음(MenuGroup.of("추천메뉴"), "/api/menu-groups");
+        소고기한우 = MenuAcceptanceTest.상품_등록되어있음(Product.of("소고기한우", 30000), "/api/products");
+        메뉴 = MenuAcceptanceTest.메뉴_등록되어있음("소고기+소고기", 50000, 추천메뉴.getId(), Arrays.asList(MenuProduct.of(소고기한우.getId(), 2L)));
+        테이블1 = 테이블_등록되어_있음(OrderTable.of(4, false), "/api/tables");
+        주문1 = OrderAcceptanceTest.주문_생성됨(테이블1.getId(), Arrays.asList(OrderLineItem.of(메뉴.getId(), 2)));
+    }
+
+    @DisplayName("테이블 관리")
+    @Test
+    void handleTable() {
+        // 테이블 생성
+        OrderTable 테이블2 = 테이블_등록되어_있음(OrderTable.of(10, false), "/api/tables");
+        Order 주문2 = OrderAcceptanceTest.주문_생성됨(테이블2.getId(), Arrays.asList(OrderLineItem.of(메뉴.getId(), 3)));
```

Comment:
```suggestion
        OrderTable 10번_테이블 = 테이블_등록되어_있음(OrderTable.of(10, false), "/api/tables");
        Order 10번_테이블_소고기_주문 = OrderAcceptanceTest.주문_생성됨(테이블2.getId(), Arrays.asList(OrderLineItem.of(메뉴.getId(), 3)));
```

1, 2로 구분하기 보다는 이름을 통해 의미를 추가해보면 어떨까요?

Reply:
피드백 감사합니다!